js/lib/bootbox.js

```diff
@@ -0,0 +1,985 @@
+/**
```

Comment:
I think that we should minify/uglify all libs on the build phrase as I proposed here:
https://github.com/IIIF/mirador/issues/1027
not in the lib folder, because it is easier to debug